packages/mephisto-task/src/index.js

```diff
@@ -79,7 +97,10 @@ const useMephistoTask = function () {
     if (taskConfig.block_mobile && isMobile()) {
       setState({ blockedReason: "no_mobile" });
     } else if (!state.isPreview) {
-      requestAgent().then((data) => afterAgentRegistration(data));
+      requestAgent().then((data) => {
+        console.log(data);
```

Comment:
We should remove console.log. Is there a logger that can be used instead?

Reply:
Yeah, I probably was just checking something and forgot to remove it.